### PR TITLE
Move schematics to datasheets.raspberrypi.org

### PR DESCRIPTION
**Pull request by aallan**
This will be over soon?

### CODE REVIEW

File: hardware/computemodule/schematics.md [hunk 1]
```diff
@@ -10,19 +10,20 @@ There is also a KiCAD PCB design set available:
 
 Schematics for the Compute Module camera/display adapter board (CMCDA):
 
-* [CMCDA Rev 1.1](schematics/rpi_SCH_CMCDA_1p1.pdf)
+* [CMCDA Rev 1.1](https://datasheets.raspberrypi.org/cmcda-schematics.pdf)
+
+schematics/rpi_SCH_CMCDA_1p1.pdf)
```
Comment (lurch) [inline hardware/computemodule/schematics.md:15]: Uh?

Comment (aallan) [reply hardware/computemodule/schematics.md:15]: Already fixed on master.

**Comment by lurch** [comment]
All the URLs you've added in this PR seem to be missing the "subdirectory" part? :confused:

**Comment by aallan** [comment]
> All the URLs you've added in this PR seem to be missing the "subdirectory" part? 😕

Fixed on master, 8f74fae. One of those days huh? :(

**Pull request merged**